### ISSUE TITLE
Chanjo inte laddad?

### ISSUE THREAD

**Issue by 47KW** [Clinical-Genomics/chanjo#225]
Får inte upp coverage (alternaivt inte heller calls) för CFAP43
https://scout.scilifelab.se/cust002/F0037468
namnändring har skett av genen påpekas

**Comment by moonso** [comment]
Yes I agree that it is really time for Chanjo, it will require some efforts though. I suggest to make a project suggestion here https://github.com/Clinical-Genomics/project-planning/issues/ so we can prioritise along with all the other stuff

**Comment by northwestwitch** [comment]
I've created a new associated project to this in [project-planning](https://github.com/Clinical-Genomics/project-planning/issues/81). Who should be in charge of it?
